packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -188,6 +188,19 @@ const SidebarNavigation: React.FC = () => {
           >
             Illustrations
           </SidebarAnchor>
+          <SidebarAnchor
```

Comment:
praise: it wasn't clear to me where this page should go, but of all the places in our IA, this one makes the most sense!

Reply:
🥳 Cheers to Simon, Katie, and the rest of the team that helped with this.